{{cookiecutter.project_slug}}/docker-compose.local.yml

```diff
@@ -95,6 +102,41 @@ services:
       - '5555:5555'
     command: /start-flower
 
+  {%- endif %}
+  {%- if cookiecutter.use_django_rq == 'y' %}
+
+  valkey:
+    image: docker.io/valkey/valkey:8.0
+    container_name: {{ cookiecutter.project_slug }}_local_valkey
+    volumes:
+      - {{ cookiecutter.project_slug }}_local_valkey_data:/data
```

Comment:
I know we're discussing migrating our Redis setup to Valkey, but we should perhaps stick to Redis for now. I would like to avoid us being in a situation where have different datastores, depending on which background queue framework was used.